mesa/Person.py

```diff
@@ -511,6 +511,18 @@ def p_fertility(self):
         else:
             return 0
 
+    def p_mortality(self):
```

Comment:
I know it's different from netlogo, but I started to fear these places where we would let the code fail silently - if we are lacking a row in the mortality table, it's not ok that we kill people of that age and go ahead silently. I'd suggest adding 1 only for ages over the top in the table and letting the code fail otherwise. Or just letting it fail in all cases - there just shouldn't be people that old around.

Reply:
> Consider it an approve - if the change in the comment takes too long, just put a TODO there?

by the way, why are checks failing?

Reply:
I added an exception, I think it will be important to do error handling PR in the future. Tests fail because agents can now die.